app/Models/Device.php

```diff
@@ -155,7 +155,7 @@ public function isUnderMaintenance()
                     $query->where('alert_schedulables.alert_schedulable_id', $this->device_id);
                 });
 
-                if ($this->groups) {
+                if (count($this->groups)) {
```

Comment:
`if ($this->groups)` is always true since it's a collection

Reply:
$this-groups->isNotEmpty() would probably be better then :)